NOTICE-binary

```diff
@@ -55,16 +55,16 @@ This product includes software from the Spring Framework,
 under the Apache License 2.0 (see: StringUtils.containsWhitespace())
 
 Apache Commons Collections
-Copyright 2001-2008 The Apache Software Foundation
+Copyright 2001-2015 The Apache Software Foundation
 
 Apache Log4j SLF4J Binding
-Copyright 1999-2019 The Apache Software Foundation
+Copyright 1999-1969 The Apache Software Foundation
```

Comment:
why 1969?

Reply:
That's weird! This part was generated. I'll go over all the end dates and update accordingly.